content/docs/instrumenting/exposition_formats.md

```diff
@@ -13,9 +13,160 @@ establish the actual format to use. A server will prefer receiving the
 protocol-buffer format, and will fall back to the text-based format if the
 client does not support the former.
 
-For details on each format and the content negotiation options, see the
-[Client Data Exposition Format](https://docs.google.com/document/d/1ZjyKiKxZV83VI9ZKAXRGKaUKK2BIWCT7oiGBKDBpjEY/edit?usp=sharing)
-document.
-
 The majority of users should use the existing [client libraries](/docs/instrumenting/clientlibs/)
 that already implement the exposition formats.
+
+## Format version 0.0.4
+
+This is the current metrics exposition format version.
+
+As of this version, there are two alternate formats understood by Prometheus: a
+protocol-buffer based format and a text format. Clients must support at least
+one of these two alternate formats.
+
+In addition, clients may optionally expose other text formats that are not
+understood by Prometheus. They exist solely for consumption by human beings and
+are meant to facilitate debugging. It is strongly recommended that a client
+library supports at least one human-readable format. A human-readable format
+should be the fallback in case the HTTP `Content-Type` header is not understood
+by the client library. The version `0.0.4` text format is generally considered
+human readble, so it is a good fallback candidate (and also understood by
+Prometheus).
+
+### Protocol buffer format
+
+* Inception: April 2014
```

Comment:
The problem is that the code for supporting the version is in `client_golang`, which wasn't vendored back then, and also didn't have a version. So the coupling between the server and the library was very loose. But I could just say that it was supported since Prometheus version 0.4.0, which is the first released version with changelog, and which came out around time time... (https://github.com/prometheus/prometheus/blob/master/CHANGELOG.md#040--2014-04-17)